pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -54,10 +54,10 @@
 
 
 class IBMQEmulatorBackend(Backend):
-    """A backend which uses the ibmq_qasm_simulator to emulate the behaviour of
-    IBMQBackend. Performs the same compilation and predicate checks as IBMQBackend.
-    Requires a valid IBMQ account.
-
+    """A Backend which uses the ibmq_qasm_simulator to emulate the behaviour of
+    IBMQBackend. Identical to :py:class:`IBMQBackend` except there is no `monitor`
+    parameter. Performs the same compilation and predicate checks as IBMQBackend.
+    Requires a valid IBM account.
```

Comment:
What is the difference of an IBM and an IBMQ account? Should we update this everywhere?